app/frontend/components/shared/base/footer.tsx

```diff
@@ -14,16 +14,17 @@ export const Footer = observer(() => {
   } = useMst()
   const { currentUser } = userStore
   const { t } = useTranslation()
-  const excludeFooterRoutes = [
+  const onlyShowFooterOnRoutes = [
     "/reset-password",
     "/accept-invitation",
     "/login",
     "/forgot-password",
     "/register",
     "/permit-applications",
```

Comment:
This will show footer for all permi-application routes and subroutes. Do we want that?

Reply:
hmm no we don't, not sure why thats there :(